calculator.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no, target-densitydpi=medium-dpi"/> 
+    <title>Blue Calculator</title>
+    <style>
+      table {
+       border: 2px solid black; 
+       margin: 10px;
+       width: 100px;
+       padding: 20px;
+       border-collapse: collapse;
+      }
+      
+      input {
+        text-align: right;
+        border: none;
+      }
+
+      input:focus {
+        outline: none;
+      }
+
+      th, td {
+       border: 1px solid black;
+       padding: 5px 10px;
+       text-align: center;
+      }
+
+      .a1{
+        background-color: gray;
+        color: white;
+      }
+
+      .a2{
+        background-color: whitesmoke;
+      }
+
+      .a3{
+        background-color: orange;
+        color: white;
+      }
+      
+    </style>
+</head>
+<body>
+  <table class="tb">
+    <tr>
+      <td class="a1" colspan="4">
+        <input class="a1" type="text" id="display">
+      </td>
+    </tr>
+    <tr>
+      <td class="a2" onclick="reset()" colspan="3">C</td>
+      <td class="a3" onclick="add('/')">%</td>
+    </tr>
+    <tr>
+      <td class="a2" onclick="add(7)">7</td>
+      <td class="a2" onclick="add(8)">8</td>
+      <td class="a2" onclick="add(9)">9</td>
+      <td class="a3" onclick="add('*')">*</td>
+    </tr>
+    <tr>
+      <td class="a2" onclick="add(4)">4</td>
+      <td class="a2" onclick="add(5)">5</td>
+      <td class="a2" onclick="add(6)">6</td>
+      <td class="a3" onclick="add('-')">-</td>
+    </tr>
+    <tr>
+      <td class="a2" onclick="add(1)">1</td>
+      <td class="a2" onclick="add(2)">2</td>
+      <td class="a2" onclick="add(3)">3</td>
+      <td class="a3" onclick="add('+')">+</td>
+    </tr>
+    <tr>
+      <td class="a2" onclick="add(0)" colspan="2">0</td>
+      <td class="a2" onclick="add('.')">.</td>
+      <td class="a3" onclick="calculate()">=</td>
+    </tr>
+  </table>
+  <script>
+    //식이아닌 문자열입력방지장치 : 이전눌렀던 값이 숫자이면 true, 아니면 false;
+    var numberClicked = false; //전역변수
+
+    function add(char) {
+
+          //숫자나 연산자 버튼을 누르면, add함수가 호출되고, 사용자가 누른값이 char에 들어감
+      if (numberClicked == false) {
```

Comment:
```suggestion
      if (!numberClicked) {
```
```suggestion
      if (numberClicked === false) {
```
`==`의 경우 
저는 코드를 작성할 때 자바스크립트의 경우 조건문 판단에 `==`(동등 연산자)보다 정확한 판단을 위해서 `===`(일치) 연산자를 통해서 판단하는 경우가 많더라구요. `==`의 경우 값이 동등한지만 비교하기 때문에 타입을 강제적으로 변경해서 타입이 다르더라도 값만 같다면 조건에 맞다고 판단할 수 있어요.

엄격하게 비교하는 `===`를 추천드리고 맨 위의 제안처럼 변수 앞에 느낌표를 통해서 작성할 수도 있어요!